### ISSUE TITLE
An instruction relies on sensory characteristics

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wai-wcag-quickref#84]
The text "Changes in this panel are reflected on the right:" fails WCAG SC 1.3.3 Sensory Characteristics, because the word "right" refers only to the visual layout of the page.

Suggested edit: "Changes in this panel are reflected in the main content area."

**Comment by yatil** [comment]
Duplicate of #72
